ros_comm/package.xml

```diff
@@ -1,6 +1,6 @@
 <package>
   <name>ros_comm</name>
-  <version>1.14.3</version>
+  <version>1.14.4</version>
```

Comment:
Sorry, I should have been more specific. Please revert this change and instead modify the file `clients/roscpp/package.xml` and replace these two lines:

```
<build_depend>rosconsole</build_depend>
<run_depend>rosconsole</run_depend>
```

with:

```
<build_depend version_gte="1.13.8">rosconsole</build_depend>
<run_depend version_gte="1.13.8">rosconsole</run_depend>
```

This basically expresses that `roscpp` needs a newer version of `rosconsole` since it requires the new function to exist. [CI will fail until a new release of `rosconsole` is being made.]